### PR TITLE
Feature/welcome page

### LINKED ISSUE #49: Add welcome page
As a user, I should see a welcome page with button to direct me to next page

### LINKED ISSUE #50: Add test for welcome page
There should be unit and integration test for welcomePage component

### PR DESCRIPTION
**Pull request by okalimbet**
#### What's this PR do?
Add a welcome page showing app title, tagline and redirect button to a homepage
Add link to a button 
Add animation to the elements on the page to improve UX
Add a home page button image 

#### What does it fix?
N/A

#### Where should the reviewer start?
App.test.js line 22
PostingView.test.js line 2
PostingView/index.js line 79
WelcomePage.scss line 1
WelcomePage.test.js line 1
WelcomePage/index.js line 1

#### How should this be manually tested?
By running `npm start' and 'npm test' all test should pass and the landing page of the website should start with WelcomePage component

#### Screenshots (if appropriate):
Screen Shot 2021-01-11 at 11 07 45 PM

### CODE REVIEW

File: src/Components/App/App.test.js [hunk 1]
```diff
@@ -119,12 +108,12 @@ describe("App", () => {
...
 
     await waitFor(() => expect(screen.queryByTestId('posting-view-element')).not.toBeInTheDocument());
-    await waitFor(() => expect(history.location.pathname).toBe("/"));
+    await waitFor(() => expect(history.location.pathname).toBe("/postings"));
   })
 
   it("When enters keyword in search it finds all events with this keyword", async() => {
     render(
-      <MemoryRouter>
+      <MemoryRouter initialEntries={['/postings']}>
```
Comment (coopterrones) [inline src/Components/App/App.test.js:116]: just movements ?

File: src/Components/App/index.js [hunk 2]
```diff
@@ -53,22 +54,45 @@ const App = () => {
...
-        render={() => (
-          <Postings
-            isSorted={isSorted}
-            postings={queriedPostings.length ? queriedPostings : postings}
-            searchByKeyWord={searchPostings}
-            sortPostingsByDate={sortPostingsByDate}
-            filterByCategory={filterPostings}
-          />
-        )}
+       <Route
```
Comment (coopterrones) [inline src/Components/App/index.js:57]: This welcome animation is dope.. Nice touch

File: src/Components/App/index.js [hunk 3]
```diff
@@ -53,22 +54,45 @@ const App = () => {
...
-            isSorted={isSorted}
-            postings={queriedPostings.length ? queriedPostings : postings}
-            searchByKeyWord={searchPostings}
-            sortPostingsByDate={sortPostingsByDate}
-            filterByCategory={filterPostings}
-          />
-        )}
+       <Route 
+        exact path="/" 
+        component={WelcomePage}
```
Comment (yiranli624) [inline src/Components/App/index.js:59]: Nice use of Route!

File: src/Components/WelcomePage/WelcomePage.scss [hunk 4]
```diff
@@ -0,0 +1,158 @@
...
+    transform: scale(1.1);
+  }
+
+  &:hover p {
+    opacity: 1;
+    transition: all 0.2s ease-in;
+  }
+}
+
+@keyframes slideInFromLeft {
```
Comment (coopterrones) [inline src/Components/WelcomePage/WelcomePage.scss:85]: Yay for keyframes. These look great

File: src/Components/WelcomePage/WelcomePage.scss [hunk 5]
```diff
@@ -0,0 +1,158 @@
+@import "../../styles/variables";
```
Comment (yiranli624) [inline src/Components/WelcomePage/WelcomePage.scss:1]: Animation looks great for our App!

**Review by yiranli624 — APPROVED** [review]

**Pull request merged**